i18n/locales/en.json

```diff
@@ -2,6 +2,7 @@
   "components.footer.scrollToTop.button": "Scroll to top",
   "components.header.links.home": "Home",
   "components.header.links.about": "About",
+  "components.header.links.learn": "Learn",
   "components.header.links.download": "Downloads",
   "components.header.links.docs": "Docs",
```

Comment:
```suggestion
  "components.header.links.docs": "API Docs",
```

Reply:
You can also add a new key (I think it would be cleaner)

Reply:
I checked several big sites - nextjs, vue, angular. They all call it docs, so I think better to keep it as is.

Reply:
Yeah you have true nextjs call their section docs because it's an docs. Our section is api docs.
Why
Because we have two methods to see content one in page/markdown (docs) and see in Json (api) so we call that API Docs

Reply:
Got your point